src/test/java/com/marizueva/laboratory/hw5/pages/DifferentElementPage.java

```diff
@@ -0,0 +1,59 @@
+package com.marizueva.laboratory.hw5.pages;
+
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class DifferentElementPage extends BasePage {
+
+    public DifferentElementPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(css = "div.checkbox-row > label.label-checkbox:nth-child(1)")
+    private WebElement waterCheckbox;
+
+    @FindBy(css = "div.checkbox-row > label.label-checkbox:nth-child(3)")
+    private WebElement windCheckbox;
```

Comment:
Could be combined to the list of web elements

Reply:
fixed